phpcs.xml.dist

```diff
@@ -74,4 +74,7 @@
 			</property>
 		</properties>
 	</rule>
+	<rule ref="WordPress.PHP.NoSilencedErrors.Discouraged">
```

Comment:
How many times are we silencing errors or how many times are we using `phpcs:ignore` for this rule?

Reply:
I believe it was about 17 times in total

Reply:
This is being reported because the Sniff was renamed 
from `Generic.PHP.NoSilencedErrors.Discouraged` to `WordPress.PHP.NoSilencedErrors.Discouraged`

I'd rather see us change the name in our `phpcs:ignore` statements.

Reply:
Thanks for pointing this out again. I didn't pay attention to this comment in the issue . Sorry :(. Will update the PR.

Reply:
No problem :)